numba/cuda/compiler.py

```diff
@@ -547,6 +590,14 @@ def inspect_asm(self):
         '''
         return self._func.ptx.get().decode('ascii')
 
+    def inspect_sass(self):
```

Comment:
Should update the docs at  https://github.com/numba/numba/blame/master/docs/source/cuda-reference/kernel.rst#L60

Reply:
I've now added `inspect_sass` to the docs - thanks for catching this!